profiles/bylickilabs.json

```diff
@@ -1,6 +1,6 @@
 {
-  "image": "https://user-images.githubusercontent.com/109308073/208296882-ea239594-81ea-419f-ae6f-e3470ccc9a3e.png",
-  "issueId": 1573,
+  "image": "https://user-images.githubusercontent.com/109308073/209451382-c2c90f77-0c2a-45dc-bd06-1f5418910dd9.jpg",
```

Comment:
Why was this changed? 🤔

Reply:
I was adding my info, then I saw this file has been modified (not by me). So when I wanted to commit mine, that one was also on the commit page. So I tried to delete it🤣 It did not work, so I committed all and knew I had to explain🤩.

My apologies. 🙏🏼 Only the changes to my file should be accepted. Please restore what happened on that file🙏🏼

Reply:
Do you need any help reverting the changes?